OpenDreamRuntime/Procs/Native/DreamProcNativeWorld.cs

```diff
@@ -38,5 +38,67 @@ public static async Task<DreamValue> NativeProc_Export(AsyncNativeProc.State sta
 
             return new DreamValue(list);
         }
+
+        [DreamProc("GetConfig")]
+        [DreamProcParameter("config_set", Type = DreamValue.DreamValueType.String)]
+        [DreamProcParameter("param", Type = DreamValue.DreamValueType.String, DefaultValue = null)]
+        public static DreamValue NativeProc_GetConfig(DreamObject src, DreamObject usr, DreamProcArguments arguments)
+        {
+            var config_set = arguments.GetArgument(0, "config_set").Stringify();
+            var param = arguments.GetArgument(1, "param");
+
+            switch (config_set) {
+                case "env":
+                    if (param == DreamValue.Null) {
+                        var keys = Environment.GetEnvironmentVariables().Keys;
+                        var list = DreamList.Create(keys.Count);
+                        foreach (var key in keys) {
+                            list.AddValue(new DreamValue((string)key));
+                        }
+                        return new DreamValue(list);
+                    } else if (Environment.GetEnvironmentVariable(param.Stringify()) is string strValue) {
+                        return new DreamValue(strValue);
+                    } else {
+                        return DreamValue.Null;
+                    }
+                case "admin":
+                    throw new NotSupportedException("Unsupported GetConfig config_set: " + config_set);
+                case "ban":
+                case "keyban":
+                case "ipban":
+                    throw new NotSupportedException("Unsupported GetConfig config_set: " + config_set);
+                default:
+                    throw new ArgumentException("Incorrect GetConfig config_set: " + config_set);
+            }
+        }
+
+        [DreamProc("SetConfig")]
+        [DreamProcParameter("config_set", Type = DreamValue.DreamValueType.String)]
+        [DreamProcParameter("param", Type = DreamValue.DreamValueType.String)]
+        [DreamProcParameter("value", Type = DreamValue.DreamValueType.String, DefaultValue = null)]
+        public static DreamValue NativeProc_SetConfig(DreamObject src, DreamObject usr, DreamProcArguments arguments)
+        {
+            var config_set = arguments.GetArgument(0, "config_set").Stringify();
+            var param = arguments.GetArgument(1, "param").Stringify();
+            var value = arguments.GetArgument(2, "value");
+
+            switch (config_set) {
+                case "env":
+                    if (value == DreamValue.Null) {
+                        Environment.SetEnvironmentVariable(param, null);
+                    } else {
+                        Environment.SetEnvironmentVariable(param, value.Stringify());
```

Comment:
Shouldn't use `Stringify()` here. Byond seems to just not set the value when `value` isn't a string.